tests/test_c_parser.py

```diff
@@ -88,12 +88,15 @@ def parse(self, txt, filename=''):
     def setUp(self):
         self.cparser = _c_parser
 
-    def assert_coord(self, node, line, file=None):
+    def assert_coord(self, node, line, column, file=None):
         self.assertEqual(node.coord.line, line)
+        if column:
```

Comment:
Yes, a default value of None and a test against None should be better, I think

Starting column at 1 is fine